facades/PC/src/main/java/org/terasology/engine/splash/graphics/Color.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * The MIT License
+ *
+ * Copyright © 2015, Heiko Brumme
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.terasology.engine.splash.graphics;
+
+import org.joml.Vector3f;
+import org.joml.Vector4f;
+
+/**
+ * This class represents a RGBA color.
+ *
+ * @author Heiko Brumme
```

Comment:
Assuming this code is copy-pasted in from somewhere we should probably attribute it somehow rather than use author tags, which go against our style code. A link to some webpage covering it would be better? And of course the code license should be compatible for sure :-)

Same goes for any other similar code in this PR

Also: We already have a Color class, but I suppose they're different?